ansible.cfg

```diff
@@ -1,2 +1,4 @@
 [ssh_connection]
 pipelining = True
+[defaults]
+executable = /bin/bash
```

Comment:
@mpagot For AWS the ansible configuration needs to include this setting.  It may need to be added to the glue code that same way as pipelining was!

Reply:
Could you achieve the same by setting some environment variable? Why is it an AWS specific one? Even if, strictly speaking only AWS need it, is it fine to run Azure and CGP with it too?

Reply:
It's required for cloud detection as the command use a pipe to grep stdout from `dmidecode` to `grep`.  However, if it will be a problem I could look at getting rid of this and using ansible to do some regexp on stdout from `dmidecode` instead.

Reply:
The above code has been removed.